packages/@o3r/testing/schematics/ng-add/index.ts

```diff
@@ -1,7 +1,22 @@
-import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { apply, chain, MergeStrategy, mergeWith, move, renameTemplateFiles, Rule, SchematicContext, template, Tree, url } from '@angular-devkit/schematics';
+import { getTestFramework, getWorkspaceConfig } from '@o3r/schematics';
+import { askConfirmation } from '@angular/cli/src/utilities/prompt';
 import { NgAddSchematicsSchema } from '../../schematics/ng-add/schema';
+import { updateFixtureConfig } from './fixture';
+import type { PackageJson } from 'type-fest';
 import * as path from 'node:path';
 import * as fs from 'node:fs';
+import { NodeDependencyType } from '@schematics/angular/utility/dependencies';
```

Comment:
I think can work with `import type`

Reply:
it's actually an `enum`. I'm using `NodeDependencyType.Dev`